src/ORM/Query.php

```diff
@@ -31,6 +33,41 @@
  * into a specific iterator that will be responsible for hydrating results if
  * required.
  *
+ * @see CollectionInterface For a full description of the collection methods supported by this class
+ * @method CollectionInterface each(callable $c) Passes each of the query results to the callable
```

Comment:
Too long :(

If it turns out that ApiGen cannot figure out on its own, I will change it.